source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+**Device VPN**
+    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications. Of course even for devices provided by your business, this tends not to be the modern approach.
+
+**Per-app VPN**
+    In contrast, the more common approach is to use a per-app VPN. This provides a connection to the VPN when needed (on-demand). For example, when using a particular app.
+
+Regardless of the commonness of either option above, Mattermost provides support for both. Because Mattermost supports the AppConfig standard, both options are compatible with EMM providers.
+
+    **Note:** Will you be connecting via a corporate proxy server? If so, `review our FAQ <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`_ covering architecture, troubleshooting and best practices.
+
+----
+
+##########################################
+Choosing the Right Mobile Deployment Model
+##########################################
+
+At this point you should have read through the deployment primer. It provides a large amount of context for the principles and best practices that follow. This next section aims to help you choose one of two recommended deployment models.
+
+*****************************************
+Using the Public App Stores (Recommended)
+*****************************************
+
+**Desired Outcomes:**
+
+- Quick rollout of the Mattermost mobile application.
+- Allow users to install the application on their device.
+- Ensure a high level of security and controlled access.
+- Use existing, internal processes and tools.
+
+We recommend either of these options if you are…
+
+- Testing out the mobile applications
+- Deploying Team Edition servers using no push notifications, or push notifications from Mattermost’s `TPNS <https://docs.mattermost.com/overview/faq.html#tpns>`_ (Test Push Notification Service)
+- Deploying Enterprise Edition servers using push notifications from Mattermost’s `HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ (Hosted Push Notification Service)
+
+The mobile applications provided by Mattermost work with our hosted version of the Mattermost Push Notification Service (MPNS). This represents the easier path. The Mattermost mobile applications can be deployed with or without an EMM provider. These options are explained in more detail below. 
+
+    **Note:** HPNS is compatible with EMM providers.
+
+**Option 1** - Public App Store Installation (Easiest)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
+- Users enter URL to your hosted Mattermost Server.
+
+**Advantages:** Very easy, mobile deployment can be done by each individual user 
+**Disadvantages:** Less control over user security, no app customization
```

Comment:
Less control over user security? how? what are we saying here?

Reply:
This method puts forward no EMM provider, simply letting a user install the app. In our conversations (though perhaps I confused things). So in a case where someone wants to require a pin code, they’d need to use an EMM provider and the appropriate AppConfig settings. Is this not the case?

The documentation seems to be written in a very similar way. For example, “ As part of AppConfig, EMM administrators can set the following additional configuration options for the Mattermost mobile apps...” I’d assume the listed options aren’t otherwise available.

Reply:
"To meet other security and compliance policies, you can deploy the Mattermost apps with any EMM provider that supports AppConfig"

Reply:
If you use an EMM provider, you will have additional security controls though correct? This was called out in several interviews, that if someone is looking to extend certain security controls, like those mentioned above, this is possible. I believe it's a stretch to think any of our customers/users would believe we're trying to state we provide and insecure app.

Reply:
resolved in meeting